mitodl/micromasters
# Remove geosuggest component, revert to dropdowns

#### What are the relevant tickets?
Fixes https://github.com/mitodl/micromasters/issues/3409

#### What's this PR do?
Reverts address fields on education and employment forms to the original state and country dropdowns plus city text input.

#### How should this be manually tested?
Add or edit an education entry and employment entry on your profile page.  You should get dropdowns for state and country plus a text field for city.  Fill them in and save.  Then re-open to make sure the changes were saved correctly.
